lib/node_modules/@stdlib/assert/is-same-date-object/README.md

```diff
@@ -0,0 +1,93 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# isSameDateObject
+
+> Test if two arguments are both Date objects and represent the same date object
```

Comment:
```suggestion
> Test if two arguments are both [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) objects corresponding to the same date and time.
```

Reply:
In other places i have used the wording
 ".....are both same Date objects and resolve to the same timestamp"
 
 do you want me to update the wording as "
 
 "... both Date objects corresponding to the same date and time"

Reply:
Yes, if you don't mind. Thanks!

Reply:
alright ill do that , will make these edits and PR